scripts/install/usr/libexec/cortx-motr/motr-server

```diff
@@ -165,6 +167,13 @@ m0_server()
         fi
     fi
 
+    if [[ -n "$MOTR_LOG_PATH" && -b $MOTR_LOG_PATH ]]; then
+        log_device_path=" -L $MOTR_LOG_PATH "
+        if [[ -n "$MOTR_LOG_SIZE" ]]; then
+            log_device_size=" -V $MOTR_LOG_SIZE "
+        fi
+    fi
```

Comment:
What if it is not set, do we want to terminate the mini provisioner

Reply:
If it is not set then we go ahead as we do in confd. For ios, we won't provide log device to m0d command.